spec/views/catalog/_show_default.html.erb_spec.rb

```diff
@@ -24,4 +27,14 @@
     expect(rendered).to have_content("Alternative")
     expect(rendered).to have_content("Test")
   end
+  it "should show all other fields" do
+    expect(rendered).to have_content("Date")
+    expect(rendered).to have_content("06/17/2015")
+    expect(rendered).to have_content("Creator")
+    expect(rendered).to have_content("Brandon")
+  end
+  it "should not show blank fields" do
+    expect(rendered).to_not have_content("Material")
+    expect(rendered).to_not have_content("Space Smores")
```

Comment:
You never added Space Smores anywhere right? It showing up would be pretty crazy